loginLegacy/src/main/java/com/brandon/loginlegacy/view/LoginScreenActivity.kt

```diff
@@ -54,7 +50,7 @@ class LoginScreenActivity : AppCompatActivity() {
         }
 
         binding.createAccountRedirect.setOnClickListener {
-            navigation.navigateToCreateAccount()
+            loginViewModel.navigateToCreateAccount(this)
```

Comment:
The viewmodel should be navigating to places and the view should still be dumb, as always. The viewmodel method should be "X Clicked". Don't bleed implementations

Reply:
I fixed these and they now have better names for the view.